README.md

```diff
@@ -23,3 +23,16 @@ For now only the folowing platforms are supported:
 2. Install it in HACS
 3. Clear browser cache (no need to restart Home Assistant)
 4. Go to Configuration -> Integrations -> + -> Search for "NetDaemon"
+
+## Use
```

Comment:
Shouldn't this be documented on <https://netdaemon.xyz/> instead of the integration README?
@helto4real ?

Reply:
Actually, this is properly documented already <https://netdaemon.xyz/docs/api/api_state#set-state-of-custom-entities>
That is how you should interface with this from your ND code

Reply:
Ah - that makes more sense when read in its entirety. The first link on `SetState` says that if you want the entries to be persisted then you need to use the integration, but the second link provides more detail saying that the supported domains *are* persisted.
I'll cancel this PR but I think the main docs could be a little clearer - I think it just needs a minor update to the `SetState` page, which I'll propose.
Thanks